src/dmd/root/outbuffer.d

```diff
@@ -18,6 +18,8 @@ import core.stdc.string;
 import dmd.root.rmem;
 import dmd.root.rootobject;
 
+debug = stomp; // flush out dangling pointer problems by stomping on unused memory
```

Comment:
I guess you want to remove this before we merge ?

Reply:
yes, now that it's working